tests/translator/input/implicit_api_with_timeout.yaml

```diff
@@ -0,0 +1,27 @@
+Resources:
+  RestApiFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://sam-demo-bucket/todo_list.zip
+      Handler: index.restapi
+      Runtime: nodejs12.x
```

Comment:
Super nitpick: nodejs12 is deprecated

Reply:
ah I'll note that when writing future tests